frontend/src/pages/GraphPF/components/groupExpanded.tsx

```diff
@@ -28,6 +28,8 @@ import {
   NodeShape,
   hullPath
 } from '@patternfly/react-topology';
+import { PFColors } from 'components/Pf/PfColors';
+import { keyframes } from 'typestyle';
 
 // This is a copy of PFT DefaultGroupExpanded (v4.68.3), then modified.  I don't see a better way to really
```

Comment:
This comment needs to be updated to add `isFocused` and to remove the `element.data.` prefixes.

Reply:
formatting is off. Not a blocker.